swagger/swagger.json

```
@@ -358,9 +367,10 @@
     },
     "/v1/directdebit/payments/{paymentId}" : {
       "get" : {
+        "tags" : [ "Direct Debit" ],
         "summary" : "Find direct debit payment by ID",
```

Sorry, think I missed this one last time!

```suggestion
        "summary" : "Find Direct Debit payment by ID",
```

---

Thanks, I've fixed